agent/src/beerocks/monitor/mon_wlan_hal_loader.cpp

```diff
@@ -7,7 +7,7 @@
  */
 
 // Monitor HAL
-#include <beerocks/bwl/mon_wlan_hal.h>
+#include <mon_wlan_hal.h>
```

Comment:
For me, it is more readable having the submodule prefix in the include, we try to keep this convention throughout the code.
We could keep it as `#include <bwl/mon_wlan_hal.h>` and move the public headers from `bwl/common` to `bwl/include/beerocks/bwl` like we do in the framework.

What do you think?

Reply:
I agree, but I would do such cleanups in later commits.